static_file_handler.cc

```diff
@@ -56,16 +62,30 @@ StaticFileHandler::Status StaticFileHandler::HandleRequest(const Request& reques
     if (ReadFile(file_path, file_content)) {
         response->SetStatus(Response::ResponseCode::OK);
 
-        std::string content_type = GetMimeType(file_path);
+        bool isMarkdown = false;
+        std::string content_type = GetMimeType(file_path, isMarkdown);
```

Comment:
Yeah it would be definitely cleaner that way... The main reason I did it the way I did it is that a separate IsMarkdown(file_path) function will be almost an exact replica of GetMimeType(), since we need to first get the extension, which is like half the function. What do you think?